spec/setup.js

```diff
@@ -114,7 +114,7 @@ const runTest = () => {
   const jasminePath = path.join(
     __dirname, '..', 'node_modules', '.bin', 'jasmine');
   const nycPath = path.join(
-    __dirname, '..', 'node_modules', 'nyc', 'bin', 'nyc.js');
+    __dirname, '..', 'node_modules', '.bin', 'nyc');
```

Comment:
Modified to use the nyc binary since this was giving some issues with windows (it was opening the nyc file).

Reply:
Update with master - I made this change to fix CI tests.

Reply:
Done.